framework/commandParser.js

```diff
@@ -297,6 +297,14 @@ class CommandParser {
             );
           }
         }).catch(e => {
+          message.channel.send(utils.getRichEmbed(
```

Comment:
good idea for users. would want to make it conditional so that users can turn it off or make sure sensitive information isn't leaked to users.

Reply:
Yes, makes sense, I'll probably use the config.json to do that